website/source/docs/agent/watches.html.md

```diff
@@ -266,26 +266,48 @@ An example of the output of this command:
 
 The "service" watch type is used to monitor the providers
 of a single service. It requires the "service" parameter
-and optionally takes the parameters "tag" and "passingonly".
-The "tag" parameter will filter by tag, and "passingonly" is
-a boolean that will filter to only the instances passing all
-health checks.
+and optionally takes the parameters "tag", "tags", and 
```

Comment:
Are the docs here still accurate after your latest updates?

Reply:
Ah!  my bad, i updated the top statement but forgot about that one.  Apologies!

Reply:
No need to apologize. Thanks for all your work on this.

Reply:
@mkeeler fixed.